src/ngraph/frontend/onnx_import/utils/arg_min_max_factory.hpp

```diff
@@ -0,0 +1,58 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#pragma once
+
+#include <memory>
+
+#include "core/node.hpp"
+#include "default_opset.hpp"
+#include "ngraph/node.hpp"
+#include "ngraph/op/topk.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace arg
+        {
+            /// \brief          Factory class which generates sub-graphs for ONNX ArgMin, ArgMax
+            /// ops.
```

Comment:
[NON-BLOCK]
It looks weird can we remove some of these spaces?

Reply:
Done